src/test/java/org/folio/logintest/RefreshTest.java

```diff
@@ -0,0 +1,209 @@
+    PostgresClient.getInstance(vertx);
+
+    var cookieHeader = LoginAPI.ACCESS_TOKEN + "=321;" + LoginAPI.REFRESH_TOKEN + "=123";
+    var cookieHeaderExpired = LoginAPI.REFRESH_TOKEN + "=abc;" + LoginAPI.ACCESS_TOKEN + "=expiredtoken";
+    var cookieHeaderMissingAccessToken = LoginAPI.REFRESH_TOKEN + "=123;";
+    var cookieHeaderDuplicateKey = LoginAPI.ACCESS_TOKEN + "=xyz;" + LoginAPI.REFRESH_TOKEN + "=xyz;" + LoginAPI.ACCESS_TOKEN + "=xyz";
+
+    spec = new RequestSpecBuilder()
+        .setBaseUri("http://localhost:" + port)
+        .addHeader(XOkapiHeaders.URL, "http://localhost:" + mockPort)
+        .addHeader(XOkapiHeaders.TENANT, TENANT_DIKU)
+        .addHeader("Cookie", cookieHeader)
+        .build();
+
+    specExpiredToken = new RequestSpecBuilder()
+        .setBaseUri("http://localhost:" + port)
+        .addHeader(XOkapiHeaders.URL, "http://localhost:" + mockPort)
+        .addHeader(XOkapiHeaders.TENANT, TENANT_DIKU)
+        .addHeader("Cookie", cookieHeaderExpired)
+        .build();
+
+    specBadRequestMissingAccessTokenCookie = new RequestSpecBuilder()
+        .setBaseUri("http://localhost:" + port)
+        .addHeader(XOkapiHeaders.TENANT, TENANT_DIKU)
+        .addHeader("Cookie", cookieHeaderMissingAccessToken)
+        .build();
+
+    specBadRequestEmptyCookie = new RequestSpecBuilder()
+        .setBaseUri("http://localhost:" + port)
+        .addHeader(XOkapiHeaders.TENANT, TENANT_DIKU)
+        .addHeader("Cookie", "")
+        .build();
+
+    specBadRequestNoCookie = new RequestSpecBuilder()
+        .setBaseUri("http://localhost:" + port)
+        .addHeader(XOkapiHeaders.TENANT, TENANT_DIKU)
+        .build();
+
+    specDuplicateKeyCookie = new RequestSpecBuilder()
+        .setBaseUri("http://localhost:" + port)
+        .addHeader(XOkapiHeaders.TENANT, TENANT_DIKU)
+        .addHeader("Cookie", cookieHeaderDuplicateKey)
+        .build();
+
+    TenantAttributes ta = new TenantAttributes().withModuleTo("mod-login-1.1.0");
+    vertx.deployVerticle(Mocks.class.getName(), mockOptions)
+        .compose(res -> vertx.deployVerticle(RestVerticle.class.getName(), options))
+        .compose(res -> TestUtil.postSync(ta, TENANT_DIKU, port, vertx))
+        .onComplete(context.asyncAssertSuccess());
+  }
+
+  @Test
+  public void testRefreshCreated(final TestContext context) {
+    RestAssured.given()
+        .spec(spec)
+        .when()
+        .post(REFRESH_PATH)
+        .then()
+        .log().all()
+        .statusCode(201)
+        .contentType("application/json")
+        .cookie(LoginAPI.REFRESH_TOKEN, RestAssuredMatchers.detailedCookie()
+            .value(Mocks.REFRESH_TOKEN)
+            // Account for time drift because we're using Now.Instant to compute max-age.
+            .maxAge(allOf(greaterThan(Mocks.REFRESH_TOKEN_EXPIRATION - 2), lessThan(Mocks.REFRESH_TOKEN_EXPIRATION + 1)))
+            .path("/authn")
+            .httpOnly(true)
+            .domain(is(not(emptyOrNullString())))
```

Comment:
Please require null domain:

```
// not setting Domain disables subdomains: 
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie#domaindomain-value
.domain(is(nullValue()))
```

Setting a domain opens the cookie for all subdomains, for example hacker.okapi.folio.example.com. This is not what we want.

Not setting a domain restricts the cookie for the domain only, for example okapi.folio.example.com. This is what we want.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie#domaindomain-value